Source/Scripts/SourceToBinary.cs

```diff
@@ -15,23 +15,26 @@ namespace Scripts;
 
 public class SourceToBinary {
 
-  public static Command GetCommand() {
+  public static Command GetCommand(TextWriter outputWriter) {
     var result = new Command("source-to-binary", "");
     var inputArgument = new Argument<FileInfo>("input", "Dafny source file");
     result.AddArgument(inputArgument);
-    var outputArgument = new Argument<FileInfo>("output", "File to write binary output to");
-    result.AddArgument(outputArgument);
-    result.SetHandler((file1, file2) => Handle(file1.FullName,
-      new StreamWriter(file2.FullName)),
-      inputArgument, outputArgument);
+
+    var deleteSourcesOption = new Option<bool>("--delete-sources");
```

Comment:
Option is worth a docstring even just to understand better when reading the source here.

Name makes me think we deleting source code though, perhaps `--delete-source-locations`?